test/functional/rpc_blockchain.py

```diff
@@ -388,6 +388,17 @@ def _test_gettxoutsetinfo(self):
         # Unknown hash_type raises an error
         assert_raises_rpc_error(-8, "'foo hash' is not a valid hash_type", node.gettxoutsetinfo, "foo hash")
 
+        self.restart_node(
+            0,
+            extra_args=[
+                "-coinstatsindex=1",
+                "-reindex"
+            ],
+        )
+        # Block not found error
+        assert_raises_rpc_error(-5, "Block not found", self.nodes[0].gettxoutsetinfo, "none", "00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09", True)
```

Comment:
In 341d5e1da1a40ed6dec449bc64132de5ae8788c2:
```suggestion
        assert_raises_rpc_error(-5, "Block not found", node.gettxoutsetinfo, "none", "00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09", True)
```

Reply:
thanks fixed in f169562d85b332576b3a1cf32755e43023227ed9